interchaintest/poa_group_test.go

```
@@ -92,10 +100,14 @@ func TestGroupPOA(t *testing.T) {
 	require.NoError(t, err)
 
 	groupGenesis := createGroupGenesis()
+	wasmGenesis := append(groupGenesis,
```

golangci-lint (1.62.2)

appendAssign: append result not assigned to the same slice

(gocritic)